Package.swift

```diff
@@ -39,13 +39,13 @@ let package = Package(
             name: "KnitBuildPlugin",
             capability: .buildTool,
             dependencies: [
-                .target(name: "KnitCommand"),
+                .target(name: "knit-cli"),
             ]
         ),
 
         // MARK: - CLI
         .executableTarget(
-            name: "KnitCommand",
+            name: "knit-cli",
```

Comment:
Oof that's a bummer, i remember making these different cause of past ambiguity issues between a product and target with the same name. But if it fixes issues with Xcode 16 it seems worth it